tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -405,7 +554,7 @@ def _load_tracking_segment(
 
 
 def _generate_test_data(image_subdir: str, _: str, __: _DATA_GETTER) -> Iterable[Data]:
-    yield from map(Data, _utility.glob(os.path.join(image_subdir, "*.jpg")))
+    yield from map(Data, glob(os.path.join(image_subdir, "*.jpg")))
```

Comment:
this function is called only once, also there is only one line, is it necessary to write a function?

Reply:
This function is used for reusing `_load_tracking_segment` for MOT and MOTS